discv5/discv5-wire.md

```diff
@@ -263,14 +163,14 @@ PONG is the reply to PING.
 
 ### FINDNODE Request (0x03)
 
-    message-data = [request-id, distance]
+    message-data = [request-id, distance₁, distance₂, ..., distanceₙ]
```

Comment:
This would be much easier to implement in our RLP framework as `[request-id, [distance₁, distance₂, ..., distanceₙ]]`

Also has the nice benfit of all of the message formats having a fixed number of elements at their base level.

Reply:
I'm OK with it if you must have it this way. Actually, now that you bring it up, we could probably get away without the outer list. That'd be a more radical change though. What do you think?

Reply:
> we could probably get away without the outer list. That'd be a more radical change though. What do you think?

I don't understand what you are suggesting here.

Reply:
I mean

```
message-data = request-id || [distance₁, distance₂, ..., distanceₙ]
```